### ISSUE TITLE
Blob missing bracket in modelContainer

### ISSUE THREAD

**Issue by dtadic** [Raizlabs/DBFlow#331]
in Model, this:
``` Java
    @Column(name = "image_blob")
    Blob imageBlob;
```
generates this in Model$Container
``` Java
    int indeximage_blob = cursor.getColumnIndex("image_blob");
    if (indeximage_blob != -1)  {
      if (cursor.isNull(indeximage_blob)) {
        modelContainer.put("image_blob",new com.raizlabs.android.dbflow.data.Blob(null);
      } else {
        modelContainer.put("image_blob",new com.raizlabs.android.dbflow.data.Blob(cursor.getBlob(indeximage_blob));
      }
    }
```
missing last bracket in both modelContainer.put() calls

**Comment by martinneff** [comment]
same problem...

**Comment by agrosner** [comment]
The annotation processor is a little too flaky for my liking. Ill fix this in `2.3.0` and am going to rewrite the processor for `3.0`

**Comment by simplysoft** [comment]
@agrosner Any updates on this? This is pretty serious bug as it prevents the whole project from compiling. If you can point us in the right direction we might be able to help you fix the problem.
BTW: for 3.0 we'd recommend using https://github.com/square/javapoet to prevent such things from happen. 

@martinneff what a coincidence :)

**Comment by agrosner** [comment]
Check out the 3.0 branch if you can. Migration to it might take some time but it has that exact change underneath. Among other massive changes to the library.

**Comment by agrosner** [comment]
3.0 is taking some time due to me wanting to ensure it is absolutely amazing and works the best ways possible. It's a significant change, possibly the largest change since the library was released.

**Comment by simplysoft** [comment]
Ok, we would love to try it out. Is there an easy way to get the 3.0 artefact?
We tried https://jitpack.io/#Raizlabs/DBFlow but it only compiles dbflow-processor and the dbflow-core artefact, the dbflow artefact is missing

**Comment by agrosner** [comment]
fixed in 3.0/develop branch. no word on jitpack support yet
